ntbs-service/Pages/Notifications/NotificationModelBase.cs

```diff
@@ -19,16 +19,19 @@ public abstract class NotificationModelBase : PageModel, IWithNotificationBanner
         protected INotificationService Service;
         protected IAuthorizationService _authorizationService;
         protected INotificationRepository NotificationRepository;
+        protected IUserHelper _userHelper;
         protected IUserService _userService;

```

Comment:
This user service was added by me for read-only user purposes so I think it can be removed now!

Reply:
I left this because it's protected and used by one of the derived classes - but you're right that it would be better to remove it from here and just have it in that one class.